lib/dpul_collections/indexing_pipeline.ex

```diff
@@ -58,11 +58,14 @@ defmodule DpulCollections.IndexingPipeline do
   Writes or updates hydration cache entries.
   """
   def write_hydration_cache_entry(attrs \\ %{}) do
+    attrs = Map.merge(%{related_data: %{}}, attrs)
```

Comment:
I added this as a default because we have tests that don't pass related_data, I don't know that it's actually real.

Reply:
The hydration_cache_entry complains that you didn't pass it.

Reply:
Ahh, I think I can do `attrs[:related_data]` instead of `attrs.related_data` and it won't raise if it's nil